templates/contest/list.html

```diff
@@ -148,8 +148,12 @@ <h4>{{ _('Active Contests') }}</h4>
             <table class="contest-list table striped">
                 <thead>
                 <tr>
-                    <th style="width:90%">{{ _('Contest') }}</th>
-                    <th>{{ _('Users') }}</th>
+                    <th style="width:90%">
```

Comment:
I am not sure if the correct behaviour is to make all the table headers clickable, since only past contests are affected.